src/Sankey/SankeyLabel/SankeyLabel.tsx

```diff
@@ -72,9 +81,15 @@ export interface SankeyLabelProps {
    * Custom formatting for the label.
    */
   format?: (value: SankeyLabelFormatProps) => any;
+
+  /**
+   * 
+   */
+  ellipsis?: number | 'none' | 'auto'
 }
 
 const LABEL_PADDING = 5;
+const LABEL_TRUNCATE_LENGTH = 10;
```

Comment:
I would make this where u can pass it in and set this as default

Reply:
The `ellipsis` option in `SankeyLabel` takes in a number as well which we then use to set the ellipsize cut-off directly and don't use any of our internal available width based logic.